### PR TITLE
UI-UX Design - Interview Resources

### PR DESCRIPTION
**Pull request by areebniyas**
**IMPORTANT:** When creating your Pull Request, please use the following format for the title: `Category - Resource Name & Section`  

PR Title samples:

- `Web development - CSS Portal & Tools`
- `Web development - Leetcode & Challenges`

### Fixes Issue

#104 

### Description: 

Adds 3 useful and reputed resourecs to prepare for UI/UX design interviews along with questions asked from UX researchers at Google, Amazon, Microsoft and Facebook.

Addresses issue #104 

### Checklist:
Before submitting your pull request, ensure that you have completed the following tasks: 

- [x] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [x] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

File: UI-UX Design/README.md [hunk 1]
```diff
@@ -276,12 +276,23 @@ Most members work in the field of UX Design or UX Research as UX Designer, Usabi
...
+        <td><a href="https://youtube.com/playlist?list=PL1wFSWLYg_Ia97VL4pv-Ovz9330HfZX_k&si=1cHzpetgv_-0uszp">Webdesign Interviews/Podcast</a></td>
+        <td>"Welcome to the 'Web Design Business Podcast Interviews' channel. On this channel, you'll find content covering the business aspects of web design, video interviews from podcast, and live stream Q&A sessions, with new videos typically posted 1-2 times a week.</td>
       </tr>
       <tr>
-      <td><a href="https://www.youtube.com/watch?v=2OzTi0iUMlE">UI/UX Design Job Interview Questions and Answers</a></td>
-      <td>Questions that you will most likely be asked in a UX design interview as well as going over tips on how best to answer them.</td>
+        <td><a href="https://www.youtube.com/watch?v=2OzTi0iUMlE">UI/UX Design Job Interview Questions and Answers</a></td>
+        <td>Questions that you will most likely be asked in a UX design interview as well as going over tips on how best to answer them.</td>
```
Comment (jfmartinz) [inline UI-UX Design/README.md:284]: Please remove this one, it exist already in the section. I will merged once you remove it. thanks

**Review by jfmartinz — CHANGES_REQUESTED** [review]

File: UI-UX Design/README.md (hunk 1 above)
Comment (areebniyas) [reply UI-UX Design/README.md]: Resolved with [74a3406](https://github.com/jfmartinz/ResourceHub/pull/108/commits/74a3406979749aeca13464bb71ea1f06bbac4794)

**Review by jfmartinz — APPROVED** [review]
Thanks @areebniyas!

**Comment by areebniyas** [comment]
@jfmartinz Could you please add the `hacktoberfest-accepted` label

**Comment by jfmartinz** [comment]
> @jfmartinz Could you please add the `hacktoberfest-accepted` label

Done. Thanks! :)

**Pull request merged**